README.md

```diff
@@ -1 +1,36 @@
-# weather-app-backend
\ No newline at end of file
+# Project Title
+Full Stack Weather App - Backend
+## Description
+The backend of the Full Stack Weather App serves as the server-side component responsible for handling requests, processing data, and interacting with the database. This Node.js/Typescript application uses Express to create RESTful APIs for retrieving weather information and forecasts from external services.
+## Technologies Used
+Node.js
+Typescript
+Express
+MongoDB
+RESTful API
+Postman
+GitHub Actions
+SonarQube
+
+## Installation
+
+1. Clone the repository:
+git clone https://github.com/AnaRaducanu/weather-app-backend.git
+
+2. Install dependencies:
+npm install
+
+3. Start the server:
+npm start
+
+4. Download and setup locally SonarQube
+
+6. To run tests, use the following command:
+npm test
+
+## Contribution Guidelines
+
+We welcome contributions to the backend of the Full Stack Weather App. To contribute, please follow these guidelines:
+Fork the repository and create a new branch for your feature.
+Submit a pull request detailing the changes made and the purpose of the feature.
+
```

Comment:
can you add also 

- how to build
- how to run as a dev, as a tester, on prod
- how to run test with test coverage
- how to run sonar
- github action info? how to use